app/controllers/users_controller.rb

```diff
@@ -91,4 +93,18 @@ def authenticate_sendgrid_webhook
       return head status: 401
     end
   end
+
+  # Send the lesson (for which a notification is being sent) to a Node
+  # server with socket.io; the Node server will then send the lesson to
+  # all mobile devices subscribed to a channel
+  def emit_lesson_notification_on_socket lesson
+    server_addr = 'https://rssf-pusher.herokuapp.com'
+    socket = SocketIO::Client::Simple.connect server_addr
+    message = lesson.as_json
+
+    socket.emit 'lessonNotification', message
+    socket.on :connect do
+      socket.emit 'lessonNotification', message
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.